### DISCUSSION TITLE
X18S doesn't turn on after update

### DISCUSSION THREAD

**Discussion by MucioJr** [FrSkyRC/ETHOS-Feedback-Community#4183] [Q&A]
I have a X18S purchased set/2023 that has been working perfectly until tomorrow morning. During the update a screen the message "your bootloader may not support autoswitching" came up. Then I tried to switch off the radio and nothing happened anymore. Radio locked and only the message usb connected appeared in the radio screen. I ended up disconnecting cables and removing the battery.
After this radio doesn't turn on anymore but I can turn the bootloader mode.
I tried to connect the radio to ethios suite and repeat de procedures but nothing happen. Radio is died...
Any help.
Ethios suite 1.5.6
Bootloader  1.4.3

![ethos main screen](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/173576869/4f71451f-cc56-4b77-ba88-c8855bf1e857)

![ethos 2](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/173576869/60c1df1c-9dc9-489c-9a87-c8aa4b20e98b)

**Comment by MucioJr** [comment]
Fixed

**Comment by bsongis-frsky** [comment]
What was the problem? Wrong firmware flashed?

**Comment by MucioJr** [comment]
Problem was as text below copied from RCGroups (see link https://www.rcgroups.com/forums/showpost.php?p=52191009&postcount=1894)



ETHOS 1.5.0. Release

[https://www.rcgroups.com/forums/show...ostcount=28272](https://www.rcgroups.com/forums/showpost.php?p=52188993&postcount=28272)


Alert: If your X18, X18S or X18SE bootloader is not Release 1.4.15 or newer make sure to flash the bootloader first before doing the 1.5.0. update. Older bootloaders are not compatible with 1.5.0. and if the firmware is flashed first the radio will not start.

1. Connect the X18SE to your PC in ETHOS mode. And select ETHOS SUITE on the radio screen.
2. Using the newest ETHOS SUITE from here [https://github.com/FrSkyRC/ETHOS-Feedback-Community/releases/tag/1.5.0](https://github.com/FrSkyRC/ETHOS-Feedback-Community/releases/tag/1.5.0)
3. Make sure SUITE shows correct radio type . Set the Branch to Stable and Version to Release 1.5.0.
4. At the bottom of the SUITE screen click on the words "WRITE OUTDATED COMPONENTS" and in the pop-up change to Write bootloader.
5. Click the down arrow.
6. The radio display should display flashing bootloader.
7. When complete the radio is ready for the 1.5.0. update.

If you do get is a situation where 1.5.0. got flashed before the bootloader was updated here is a way to recover [[https://www.rcgroups.com/forums/show...postcount=1910](https://www.rcgroups.com/forums/showpost.php?p=52193291&postcount=1910)](https://www.rcgroups.com/forums/showpost.php?p=52193291&postcount=1910)
